src/set_theory/ordinal/fixed_point.lean

```diff
@@ -1,5 +1,5 @@
 /-
-Copyright (c) 2022 Violeta Hernández Palacios, Mario Carneiro. All rights reserved.
+Copyright (c) 2018-2022 Violeta Hernández Palacios, Mario Carneiro. All rights reserved.
```

Comment:
I've never seen dates like this in mathlib...

Reply:
See [Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Copyright.20header).
```suggestion
Copyright (c) 2018 Violeta Hernández Palacios, Mario Carneiro. All rights reserved.
```